tests/durability/test_disk_failure_recovery.py

```diff
@@ -128,6 +130,7 @@ def teardown_method(self):
         LOGGER.info("Done: Teardown completed.")
 
     # pylint: disable=too-many-statements
+    @pytest.mark.durability
```

Comment:
if this is new marker, add it to pytest.ini

Reply:
Done